packages/gatsby/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@sentry/gatsby",
+  "version": "5.16.1",
+  "description": "Offical Sentry SDK for Gatsby.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/gatsby",
+  "author": "Sentry",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "node": ">=6"
+  },
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/browser": "5.16.1",
+    "@sentry/types": "5.16.1"
+  },
+  "peerDependencies": {
+    "gatsby": "*"
+  },
+  "devDependencies": {
+    "jest": "^24.7.1",
+    "npm-run-all": "^4.1.2",
+    "prettier": "^1.17.0",
+    "prettier-check": "^2.0.0",
+    "rimraf": "^2.6.3",
+    "tslint": "^5.16.0",
```

Comment:
@HazAT We can get rid of the `tslint` + `typescript` related stuff if the package doesn't use typescript files right?

Reply:
do we not need this if we export the react components?

Reply:
Ah yes good catch, if we are exporting from the react components we will still need tslint and typescript. Ignore what I said earlier then.